numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -1282,6 +1282,14 @@ typedef struct {
 #define PyArray_MultiIter_NOTDONE(multi)                \
         (_PyMIT(multi)->index < _PyMIT(multi)->size)
 
+#define PyArray_MultiIter_SIZE(multi) (_PyMIT(multi)->size)
+#define PyArray_MultiIter_NDIM(multi) (_PyMIT(multi)->nd)
+#define PyArray_MultiIter_INDEX(multi) (_PyMIT(multi)->index)
+#define PyArray_MultiIter_NUMITER(multi) (_PyMIT(multi)->numiter)
+#define PyArray_MultiIter_DIMS(multi) (_PyMIT(multi)->dimensions)
+#define PyArray_MultiIter_ITERS(multi) ((void **)_PyMIT(multi)->iters)
```

Comment:
Could we make these static functions instead?

Reply:
Done in [`1ead1e5` (#24658)](https://github.com/numpy/numpy/pull/24658/commits/1ead1e5d720f74b16ab42231c97e3dadb1c9428f)